src/components/GlobalNavigation/styles.scss

```diff
@@ -34,8 +34,7 @@
 	text-transform: uppercase;
 	width: 130px;
 
-	@media #{$wds-breakpoint-l-up} {
-		padding-right: 42px;
+	@media #{$wds-breakpoint-large-up} {
```

Comment:
Is that needed?

Reply:
removed the media query and applied to all breakpoints. to align right